### ISSUE TITLE
Documentation incorrectly names app.js instead of index.js

### ISSUE THREAD

**Issue by notronwest** [aws-amplify/docs#1090]
Starting out on mac running Mojave the instructions list creating a file in the /src/ directory named app.js. Instead it should be index.js

**Comment by sammartinez** [comment]
@notronwest Could you provide the link you are calling this out to? Is this with our Getting Started guide? Please let us know

**Comment by notronwest** [comment]
https://aws-amplify.github.io/docs/js/start?ref=amplify-js-btn&platform=purejs

**Comment by sammartinez** [comment]
@notronwest Thanks for providing the link, in looking at what is created, we chose to use `app.js` instead of `index.js` in order to have a uniformity with all frameworks we support, you'll notice if you select one of the frameworks from the getting started dropdown, Angular, Vue, React and React Native, we follow a similar pattern. I do see as well in the webpack file entry does have `entry: './src/app.js',`, are you experiencing an issue configuring the application?

**Comment by notronwest** [comment]
Thanks for looking into this. I have moved away from this portion of my
work at the moment. I plan on using Amplify on the project but I am
focusing on the core application code at the moment. When I tried to follow
the instructions exactly as they were written the test application threw an
error until I changed the name of the fie to index.js. When I get back to
this portion of the proof of concept I will look closer into why this was
happening and give you more feedback.
